doc/src/Howto_output.rst

```diff
@@ -81,6 +83,25 @@ the dimension twice (array -> scalar).  Thus a command that uses
 scalar values as input can typically also process elements of a vector
 or array.
 
+.. _disambiguation:
+
+Disambiguation
+--------------
+
+Some computes and fixes produce multiple kinds of data, e.g. a global
```

Comment:
Please use the following text instead for the new Dis section.  The disambiguation needed is only for styles, not for kinds, using the nomenclature defined immediately above.

Some computes and fixes produce data in multiple styles, e.g. a global
scalar and a per-atom vector.  Usually the context in which the input
script references the data determines which style is meant.  Example:
if a compute provides both a global scalar and a per-atom vector, the
former will be accessed by using ``c_ID`` in an equal-style variable,
while the latter will be accessed by using ``c_ID`` in an atom-style
variable.  Note that atom-style variable formulas can also access
global scalars, but in this case it is not possible to do directly because
of the ambiguity.  Instead, an equal-style variable can be defined
which accesses the global scalar, and that variable used in the
atom-style variable formula in place of ``c_ID``.

Reply:
done